src/services/activity-log/activity-log-model.js

```diff
@@ -0,0 +1,20 @@
+const mongoose = require('mongoose');
+
+const Schema = mongoose.Schema;
+const ActivityLogSchema = new Schema({
+  user: { type: String },
```

Comment:
I think from now on, shall we reorder them alphabetically?

Let's reorder all the fields in every model?